sqlmesh/core/config/base.py

```diff
@@ -123,4 +123,8 @@ def update_with(self: T, other: t.Union[t.Dict[str, t.Any], T]) -> T:
             else:
                 updated_fields[field] = getattr(other, field)
 
-        return self.copy(update=updated_fields)
+        updated = self.copy()
```

Comment:
Can you please provide more context / leave a comment around why this has been changed?

Reply:
Assignment to trigger pydantic validators. copy(update=...) did not trigger the validators.